web-client/src/ustc-ui/Icon/CaseIcons.tsx

```diff
@@ -0,0 +1,42 @@
+import { ConsolidatedCaseIcon } from '@web-client/ustc-ui/Icon/ConsolidatedCaseIcon';
```

Comment:
Rather than have this code duplicated in several spots across the trial session details pages, I pulled it out here.